sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventInstrumentation.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+
+namespace Azure.Messaging.EventHubs.Diagnostics
+{
+    internal class EventInstrumentation
```

Comment:
We should consider renaming this to `EventDataInstrumentation` to match the type that it is instrumenting.